### ISSUE TITLE
deal.II Release 9.2.0 - Open pull requests

### ISSUE THREAD

**Issue by kronbichler** [dealii/dealii#9729]
We plan for the 9.2 release on May 15, 2020. We plan to close the time window for contributions on May 8, 2020, which is when we branch off for the release and only allow bug fixes. May 8 is also the time limit I set for the 9.2 release milestone.

This issue will keep track of our progress towards the release so that everyone can adapt to the timing and we have a central place to collect the general discussion. As the release date comes closer, we will list the specific progress in our release list on #10072.

List of open pull requests and issues that need action:
- [ ] #9078 step-70 needs resources towards the end of the merge window
- [x] <s>#6625 external resources and documentation: likely postponed -> @tamiko</s>
- [x] #7922 addressed by #10108 
- [x] <s>#7428, #9047, #8603 issue with p4est: nothing we can really do about right now -> postponed for a real solution</s>
- [x] <s>#1973 regarding polynomial spaces is addressed by #8711 which is in progress, decide later whether it can be fixed in time -> address after branch to avoid risky break of interface</s>
- [ ] #9684 for step-52: @Rombur will have a look
- [x] #10006 instantiation of OpenCASCADE classes
- [x] #10086 failing OpenCASCADE test
- [x] #10008 compilation with ginkgo
- [x] #10016 compilation with symengine
- [x] #10058 convert `FEValues::dof_indices()` to `iota_view`
- [x] #7513 author must address review comments
- [x] #9558 needs documentation fix from author
- [x] #9802 needs review
- [x] #9807 needs review
- [x] #9954 author must address review comments and rebase to resolve merge conflict, will close #9597

**Comment by kronbichler** [comment]
As mentioned in #9826, we currently have somewhat too many test failures on the official `tester` machine:
https://cdash.43-1.org/viewTest.php?onlyfailed&buildid=5664
The configuration I picked is one where we should aim for zero test failures, IIRC we had 0 failures for the last release there.

**Comment by kronbichler** [comment]
We plan to branch off to the release branch on Wednesday May 13. To give some time to review the open 47 pull requests/issues and reschedule some of them, I suggest to bring up all pull requests that are not purely bug fixes that target the 9.2 release within the next 32 hours, i.e., until the end of May 8.

**Comment by kronbichler** [comment]
Apart from the pull requests that are open and either wait to be reviewed or to get the review comments fixed, the following issues with the 9.2 milestone need our attention:
- #6625 will likely get postponed, unless @tamiko makes a heroic effort or assigns someone else to do that
- #7922 - we might probably ignore it once @luca-heltai confirms, it could be that some fixes in the meantime have resolved the issue.
- #7428, #9047, #8603 all appear the same issue with p4est and nothing we can really do about right now.
- #1973 and #8711 might get postponed, depending on the progress
- #9684: @Rombur will look into
- #10006 - must look into
- #10008 - must look into
- #10016 - must look into
- #10086 - must look into
- #10058 needs to be tackled after #10018 is merged - I will merge the latter this afternoon
- #9078 / step-70 will need resources towards the end of the merge window

**Comment by luca-heltai** [comment]
#7922 - I confirm it is an issue. I opened a PR to fix it (it was an easy fix). This will make all codes that use mapping + compute point locations + particles much faster.

**Comment by kronbichler** [comment]
I will now post updates in the top post with a tick list. Feel free to add topics there or to tick them once we are done with a topic.

**Comment by tamiko** [comment]
@kronbichler I will work on #6625 after our current release when I package for debian/ubuntu.

**Comment by tamiko** [comment]
Closing now in favor of #10072
